Misc/NEWS.d/next/Library/2020-05-02-01-01-30.bpo-29753.n2M-AF.rst

```diff
@@ -0,0 +1,2 @@
+Fixes packed bitfields not being calculated properly in some cases.
+Fixes packed bitfileds not shrinking the first item.
```

Comment:
I would re-write this to give users more context. Think about the person reading the NEWS for Python 3.10. This notice doesn't explain which bitfields were not being calculated properly.

```suggestion
In ctypes, now packed bitfields are calculated properly and packed bitfields avoid shrinking the first item.
```

Reply:
Thanks.

> packed bitfields avoid shrinking the first item

It's the opposite :stuck_out_tongue: They are now shrank, as they should. I'll take this as a failure of my initial description.